tests/conftest.py

```diff
@@ -482,3 +482,30 @@ def teardown_tftp_storage():
         assert teardown.values()[0]["rc"] == 0
 
     request.addfinalizer(teardown_tftp_storage)
+
+
+@pytest.fixture(scope="function")
+def setup_yum_content(request, ansible_module):
+    """ Setup/Teardown cusom yum repo for test_positive_content_migrate."""
+    prod_name = "Animals"
+    repo_name = "Animal_repo"
+    repo_url = "http://infra-robottelo-fixtures.infra.sat.rdu2.redhat.com:50123/fake_yum0/"
```

Comment:
@vsedmik you can add this url to conf/testfm.sample.yaml and letter add it to testfm/constants.py similar to other settings there. You will also have to update the testfm.yaml present in our gitlab.